tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -335,21 +335,28 @@ def test_fail_stale_trials() -> None:
         assert isinstance(storage, RDBStorage)
         storage.heartbeat_interval = heartbeat_interval
         storage.grace_period = grace_period
-        study = create_study(storage=storage)
+        study1 = create_study(storage=storage)
+        study2 = create_study(storage=storage)
 
-        trial = study.ask()
+        trial = study1.ask()
         storage.record_heartbeat(trial._trial_id)
         time.sleep(grace_period + 1)
```

Comment:
This is a nitpick but I'm wondering if we can delete this line.
The following sleep has an effect for both trials from `study1` and `study2`.
https://github.com/optuna/optuna/pull/2330/commits/fc2489503f54d6ca607286510540a3ba3acbb98d#diff-42166922cf20058a0456b29a4eccb65307689d7aefed170b335539b5e3ed11ebR347

Reply:
Thanks for the review! I updated as you suggested.